IFC4x3/Properties/g/GasConcentration_39aIOtx2DEa802QYg_FsGV/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GasConcentration_39aIOtx2DEa802QYg_FsGV" Name="GasConcentration" UniqueId="c9912637-ec23-4e90-8002-6a2abe3f641f" PrimaryDataType="IfcText" />
```

Comment:
Seems like a number type. Propose changing `PrimaryDataType`.

Reply:
I'll ask the team on Thirsday. Could be a number or a class (like "high"), maybe existing classifications

Reply:
If urgent let's make it text, as Michel suggested

Reply:
Text is for descriptions. Label then.

Reply:
Label OK